src/components/Logo/Logo.tsx

```diff
@@ -2,10 +2,14 @@ import React from 'react';
 import { ReactComponent as LogoImage } from '../../assets/img/logo.svg';
 import styles from './Logo.module.scss';
 import { NavLink } from 'react-router-dom';
+import { useAppSelector } from '../../redux/app/hooks';
+import authSelectors from '../../redux/features/auth/authSelectors';
 
 export default function Logo() {
+  const isLogged = useAppSelector(authSelectors.getLoggedOn);
+
   return (
-    <NavLink to={'/'}>
+    <NavLink to={isLogged ? '/library' : '/'}>
```

Comment:
So the last our decision that logged user won't be able to get to the Intro Page?